content/setting-up-tugboat/select-repo-settings.md

```diff
@@ -127,6 +127,18 @@ credentials from within the repo settings.
 
 ![Authenticate with a Docker registry](../../_images/authenticate-with-a-docker-registry-add-credentials.png)
 
+By default, when you
+[specify a public image to use in Tugboat builds](/setting-up-services/how-to-set-up-services/specify-a-service-image),
+we use a Tugboat-authenticated user to pull images from Docker Hub.
+
+If you need to pull an image that requires authentication, adding your Docker Registry Authentication credentials to the
+repository settings will use those credentials for every Docker pull made from this repository, overriding Tugboat's
+default user.
```

Comment:
We only default to using our authentication for dockerhub images. We pull from other registries anonymously, unless there is authentication configured for that registry on the Tugboat repo. That's not quite clear here.